0xProto.glyphs

```diff
@@ -20,6 +27,10 @@ value = (
 0,
 0
 );
+},
+{
+name = "Write lastChange";
+value = 0;
```

Comment:
I've researched the purpose of this parameter. Can you explain why you're including it?

Reply:
This prevents Glyphs.app from writing the "Last Change" data in each glyph.
That data is more often noise than useful information, particularly when version control system like git already has info on when things were changed.

In my case, Glyphs.app was updating all the "Last Change" info just because of my timezone or something.

Most often, this "Last Change" is changed when a glyph is modified, or when a glyph was viewed but unchanged (for example if one moved something and undoes the move).

Reply:
That makes sense 👍